pontoon/contributors/utils.py

```diff
@@ -427,24 +427,89 @@ def get_project_locale_contribution_counts(contributions_qs):
     return counts
 
 
-def get_contribution_timeline_data(user, contribution_type=None, day=None):
+def get_contribution_timeline_data(
+    user, year_shown=None, contribution_type=None, day=None
```

Comment:
This applies to all the places where this variable name is used: Why "year shown"? Why not "show year"? And if this is a boolean, why does it default to `None`?